### ISSUE TITLE
Docker hungs 

### ISSUE THREAD

**Issue by debunge** [wekan/wekan#700]
I'm trying to run docker container:

```
sudo docker run -e "MONGO_URL=mongodb://localhost:27017/wekan" -e "ROOT_URL=http://52.52.52.52" --net="host" 17f9b399c1d8
=> Starting meteor app on port:80
> Starting board-background-color migration.
> Finishing board-background-color migration.
> Starting lowercase-board-permission migration.
> Finishing lowercase-board-permission migration.
> Starting change-attachments-type-for-non-images migration.
> Finishing change-attachments-type-for-non-images migration.
> Starting card-covers migration.
> Finishing card-covers migration.
> Starting use-css-class-for-boards-colors migration.
> Finishing use-css-class-for-boards-colors migration.
> Starting denormalize-star-number-per-board migration.
> Finishing denormalize-star-number-per-board migration.
> Starting add-member-isactive-field migration.
> Finishing add-member-isactive-field migration.
```

What is happening? 
And then nothing happens.

**Comment by xet7** [comment]
What are you trying to do?

You could run Wekan and MongoDB in containers like this:

```
docker run -d --restart=always --name wekan-db mongo

docker run -d --restart=always --name wekan --link "wekan-db:db" -e "MONGO_URL=mongodb://db" -e "ROOT_URL=http://localhost" -p 8080:80 mquandalle/wekan
```

8080 port can be changed to 80, or add Let's Encrypt webserver with proxy to Wekan like this:

https://github.com/wekan/wekan/wiki/Let's-Encrypt-and-Google-Auth

**Comment by debunge** [comment]
I've tried to connect to already existing mongo instance, but nevermind. 
I've done both as you've suggested and as in wiki (instructions for test install), but I cannot connect to seemingly running containers. Could you please tell me what to do after I've done all this https://github.com/wekan/wekan/wiki/Install-Wekan-Docker-for-testing

**Comment by xet7** [comment]
@debunge 

In my example above Wekan is available at http://localhost:8080 or http://your-ip-address:8080 , for example http://192.168.1.5:8080 . It's available at local network if you:

- don't have firewall enabled on computer running Docker or
- have port like 8080 or 80 (whatever you run in commandline in place of number 8080) enabled on your firewall.

Docker containers are in internal Docker network, only Wekan container port is exposed to outside. If you need to get inside containers for example to backup or restore MongoDB data, you can find docs about it at:

https://github.com/wekan/wekan/wiki/Export-Docker-Mongo-Data

All of data that is inserted to Wekan is at MongoDB database, it's not needed to backup anything else from those containers.

**Comment by debunge** [comment]
@xet7  Yes, I understood that, but for some reason docker literally hungs up: eats 100% cpu and I cannot access the port even from local machine with w3m.

**Comment by xet7** [comment]
@debunge 

How much RAM your server or VM has?

**Comment by debunge** [comment]
It is amazon t2.micro with 1GB of ram.

**Comment by xet7** [comment]
This should be fixed now with #718 . Please test, and if there is still problems, reopen issue.
https://github.com/wekan/wekan-mongodb
